backend/src/graphql/resolver/util/sendTransactionsToDltConnector.ts

```diff
@@ -0,0 +1,82 @@
+import { IsNull } from '@dbTools/typeorm'
+import { DltTransaction } from '@entity/DltTransaction'
+import { Transaction } from '@entity/Transaction'
+
+import { DltConnectorClient } from '@/apis/DltConnectorClient'
+import { backendLogger as logger } from '@/server/logger'
+import { Monitor, MonitorNames } from '@/util/Monitor'
+
+export async function sendTransactionsToDltConnector(): Promise<void> {
+  logger.info('sendTransactionsToDltConnector...')
+  // check if this logic is still occupied, no concurrecy allowed
+  if (!Monitor.isLocked(MonitorNames.SEND_DLT_TRANSACTIONS)) {
+    // mark this block for occuption to prevent concurrency
+    Monitor.lockIt(MonitorNames.SEND_DLT_TRANSACTIONS)
+
+    try {
+      await createDltTransactions()
+      const dltConnector = DltConnectorClient.getInstance()
+      if (dltConnector) {
+        logger.debug('with sending to DltConnector...')
+        const dltTransactions = await DltTransaction.find({
+          where: { messageId: IsNull() },
+          relations: ['transaction'],
+          order: { createdAt: 'ASC', id: 'ASC' },
+        })
+        for (const dltTx of dltTransactions) {
+          try {
+            const messageId = await dltConnector.transmitTransaction(dltTx.transaction)
+            const dltMessageId = Buffer.from(messageId, 'hex')
+            if (dltMessageId.length !== 32) {
+              logger.error(
+                'Error dlt message id is invalid: %s, should by 32 Bytes long in binary after converting from hex',
+                dltMessageId,
+              )
+              return
+            }
+            dltTx.messageId = dltMessageId.toString('hex')
+            dltTx.verified = true
+            dltTx.verifiedAt = new Date()
```

Comment:
Verify comes later. This sends the transaction to the iota tangle for the time being.
![transmitTransaction](https://github.com/gradido/gradido/assets/67117805/d7728a58-1461-4825-8823-e6a3b22f24b6)

This is roughly how it will look with the verify. But I didn't want to pack everything into a PR, so this step comes later.
```suggestion
```
![verifyTransaction](https://github.com/gradido/gradido/assets/67117805/fd7ffe5e-76a9-4240-8e23-fd09fb6909dd)